### PR TITLE
PMM-5476 unskipped --pprof tests, fixed failing tests

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm-tests/pmm-2-0-bats-tests/generic-tests.bats [hunk 1]
```diff
@@ -195,8 +185,6 @@ run pmm-admin summary --skip-server
 echo "$output"
     [ "$status" -eq 0 ]
     echo "${lines[-1]}" | grep ".zip created."
-    checkZipFileContents
```
Comment (BupycHuk) [inline pmm-tests/pmm-2-0-bats-tests/generic-tests.bats]: let's keep this test.

Comment (yurkovychv) [reply pmm-tests/pmm-2-0-bats-tests/generic-tests.bats]: @BupycHuk  this test exists, but we decided to remove files count check. or you think we should check it too?

Comment (BupycHuk) [reply pmm-tests/pmm-2-0-bats-tests/generic-tests.bats]: We don't have other test with the same amount of files

Comment (yurkovychv) [reply pmm-tests/pmm-2-0-bats-tests/generic-tests.bats]: we do actually, at line 174

Comment (BupycHuk) [reply pmm-tests/pmm-2-0-bats-tests/generic-tests.bats]: you are right, my bad

**Pull request merged**